engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/EngineJobExecutionActor.scala

```diff
@@ -39,6 +39,7 @@ class EngineJobExecutionActor(replyTo: ActorRef,
                               jobStoreActor: ActorRef,
                               callCacheReadActor: ActorRef,
                               jobTokenDispenserActor: ActorRef,
+                              backendSingletonActor: Option[ActorRef],
```

Comment:
Dunno, you've set a mighty high bar with `interstitialRecombobulation`.  😛 

And actually I don't think the current name is too far off the mark, but it would perhaps benefit from at least a little documentation to describe its purpose.